app/javascript/mastodon/features/account_timeline/components/familiar_followers.tsx

```diff
@@ -0,0 +1,84 @@
+import { useEffect } from 'react';
+
+import { FormattedMessage } from 'react-intl';
+
+import { Link } from 'react-router-dom';
+
+import { fetchAccountsFamiliarFollowers } from '@/mastodon/actions/accounts_familiar_followers';
+import { AvatarGroup } from '@/mastodon/components/avatar_group';
+import type { Account } from '@/mastodon/models/account';
+import { getAccountFamiliarFollowers } from '@/mastodon/selectors/accounts';
+import { useAppDispatch, useAppSelector } from '@/mastodon/store';
+
+const AccountLink: React.FC<{ account?: Account }> = ({ account }) => (
+  <Link to={`/@${account?.username}`} data-hover-card-account={account?.id}>
+    {account?.display_name}
+  </Link>
+);
+
+const FamiliarFollowersReadout: React.FC<{ familiarFollowers: Account[] }> = ({
+  familiarFollowers,
+}) => {
+  const messageData = {
+    name1: <AccountLink account={familiarFollowers.at(0)} />,
+    name2: <AccountLink account={familiarFollowers.at(1)} />,
+    othersCount: familiarFollowers.length - 2,
+  };
+
+  if (familiarFollowers.length === 1) {
+    return (
+      <FormattedMessage
+        id='account.familiar_followers_one'
+        defaultMessage='Followed by {name1}'
+        values={messageData}
+      />
+    );
+  } else if (familiarFollowers.length === 2) {
+    return (
+      <FormattedMessage
+        id='account.familiar_followers_two'
+        defaultMessage='Followed by {name1} and {name2}'
+        values={messageData}
+      />
+    );
+  } else {
+    return (
+      <FormattedMessage
+        id='account.familiar_followers_many'
+        defaultMessage='Followed by {name1}, {name2}, and {othersCount, plural, one {# other} other {# others}}'
+        values={messageData}
+      />
+    );
+  }
+};
```

Comment:
imo it'd be better to use the [`FormattedPlural`](https://formatjs.github.io/docs/react-intl/components/#formattedplural) component for this?

Reply:
I believe `FormattedPlural` only works for very simple use cases, as there doesn't seem to be a way to define a translation id?

The `formatPlural` util has this note & I think this also applies to the component:
![image](https://github.com/user-attachments/assets/5e66b5c0-5661-47df-8661-6393dc0645aa)